### PR TITLE
Typecheck ints used as port values.

### PR DESCRIPTION
**Pull request by aqk**
Keep values representing IP ports as uint16 within our own code.

Note that `YAML` and `web.TCPSite` do not support uint16 for port values, so we use `int` in those cases.

### CODE REVIEW

**Review by AmineKhaldi — DISMISSED** [review]

**Review by AmineKhaldi — DISMISSED** [review]

File: tests/util/socket.py [hunk 1]
```diff
@@ -2,14 +2,16 @@
 import socket
 from typing import Set
 
+from chia.util.ints import uint16
+
 recent_ports: Set[int] = set()
```
Comment (altendky) [inline tests/util/socket.py:7]:
```suggestion
recent_ports: Set[uint16] = set()
```

File: tests/connection_utils.py [hunk 2]
```diff
@@ -34,6 +34,7 @@ async def disconnect_all_and_reconnect(server: ChiaServer, reconnect_to: ChiaSer
 async def add_dummy_connection(
     server: ChiaServer, self_hostname: str, dummy_port: int, type: NodeType = NodeType.FULL_NODE
 ) -> Tuple[asyncio.Queue, bytes32]:
+    dummy_port = uint16(dummy_port)
```
Comment (altendky) [inline tests/connection_utils.py:37]: It seems like it would be more consistent to push this out to the call sites?

**Review by altendky — COMMENTED** [review]
Was enough isort-preference applied that the files can be removed from the isort exception list?

**Comment by emlowe** [comment]
Rebased and revived PR - closing conversations.

**Comment by emlowe** [comment]
Close and reopen for CI

**Comment by emlowe** [comment]
Managed to get this up and running - but starting to think maybe it's not worth the hassle of the code and all the casts to make yaml happy.

**Comment by emlowe** [comment]
Closing old draft PR

**Pull request closed without merging**